ipynb/filterbiomseqs.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+
+"""
+Filter sequences from a biom table based on a fasta file
+used for AG bloom filtering
+"""
+
+# amnonscript
+
+__version__ = "1.4"
+
+import biom
+
+import argparse
+import sys
+
+
+def iterfastaseqs(filename):
+	"""
```

Comment:
I don't understand why this file is still here - I already have a cleaned up version of this file [here](https://github.com/knightlab-analyses/bloom-analyses/blob/master/bloom/filter_seqs_from_biom.py)